src/main.ts

```diff
@@ -16,6 +17,7 @@ export const ROUTES: Route[] = [
     { path: 'cart', component: CartComponent },
     { path: 'testpage', component: TestpageComponent },
     { path: 'product/:id', component: ProductDetailComponent },
+    { path: 'products', component: ProductlistComponent},
```

Comment:
Perhaps this could live in the store page instead?

Reply:
Yea so I was not in my right mind when i coded any of this